### PR TITLE
[Feature Fix] Fix failing admin tests

### PR DESCRIPTION
**Pull request by Ghalko**
Move back to using Django's `Testcase` for admin app.

### CODE REVIEW

**Comment by brianjgeiger** [comment]
Looks good. Will merge when tests pass.

**Comment by brianjgeiger** [comment]
So hopefully merging develop into this branch will fix the problems you're seeing?

**Comment by Ghalko** [comment]
This is develop.

**Comment by brianjgeiger** [comment]
Sorry, I thought something had been merged recently into develop, but I was looking at the wrong thing. Ignore me.

File: tests/factories.py [hunk 1]
```diff
@@ -718,3 +719,69 @@ def _create(cls, *args, **kwargs):
...
         return super(NodeLicenseRecordFactory, cls)._create(*args, **kwargs)
+
+
+def render_generations_from_parent(parent, creator, num_generations):
+    current_gen = parent
+    for generation in xrange(0, num_generations):
+        next_gen = NodeFactory(
+            parent=current_gen,
+            creator=creator,
+            title=fake.science_sentence(),
```
Comment (mfraezz) [inline tests/factories.py]: `fake` here is just `faker.Factory`, which doesn't normally have `.science_<things>`. This looks like it came from `create_fakes.py`, which also called `fake.add_provider(Sciencer)`.

Comment (samchrisinger) [inline tests/factories.py]: Attempted to address this in https://github.com/CenterForOpenScience/osf.io/pull/5382/commits/64532c5040cf47eb7d3a366c7d733c05b767f74f

Comment (mfraezz) [inline tests/factories.py]: Whoops, didn't see that

File: tests/factories.py [hunk 2]
```diff
@@ -718,3 +719,69 @@ def _create(cls, *args, **kwargs):
...
+        else:
+            new_parent = render_generations_from_parent(parent, creator, node_number)
+    return new_parent
+
+
+def create_fake_user():
+    email = fake.email()
+    name = fake.name()
+    parsed = impute_names(name)
+    user = UserFactory(username=email, fullname=name,
```
Comment (abought) [inline tests/factories.py]: Do we need all this boilerplate? Why can't we just use `UserFactory()` to create the instance directly? (any arguments not provided should be taken from the defaults)

**Comment by Ghalko** [comment]
Copied Sam's closed PR, all builds are green.

File: api/base/settings/defaults.py [hunk 3]
```diff
@@ -27,6 +27,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = osf_settings.DEBUG_MODE
+DEBUG_PROPAGATE_EXCEPTIONS = True
```
Comment (cwisecarver) [inline api/base/settings/defaults.py]: If you install the [django_extensions](http://django-extensions.readthedocs.org/en/latest/runserver_plus.html) package and remove this and run `python manage.py runserver_plus` you'll get werkzeug and it's interactive debugging console like you do with flask. There's also several other fancy things.

File: tests/base.py [hunk 4]
```diff
@@ -251,8 +252,7 @@ def setUp(self):
         self.app = TestAppJSONAPI(api_django_app)
 
 
-@override_settings(DEBUG_PROPAGATE_EXCEPTIONS=True)
```
Comment (cwisecarver) [inline tests/base.py]: This should be specific to the test cases or the yellow and grey exception error page with the trace turns into plain/text and everyone is :disappointed:

File: api/base/settings/defaults.py (hunk 3 above)
Comment (Ghalko) [inline api/base/settings/defaults.py]: Noted.

File: tests/base.py (hunk 4 above)
Comment (icereval) [inline tests/base.py]: @Ghalko, plz move this setting back to the decorator only so it only applies to the test cases and ping me with the pr (against 0.67.0) so i can merge, thx :octocat:

**Pull request merged**